gitgud/skills/intro/__init__.py

```diff
@@ -4,8 +4,9 @@
 from gitgud.skills.user_messages import show_tree
 
 class Intro(BasicLevel):
-    def __init__(self):
+    def __init__(self, readable_name=""):
```

Comment:
In some of the new updates, I got rid of the `__init__` method here. You'll see after merging master. We can just set everything via the constructor for `BasicLevel`